docs/_docs/20-fasterkv-basics.md

```diff
@@ -273,13 +273,14 @@ public static void Test()
 {
   using var log = Devices.CreateLogDevice("C:\\Temp\\hlog.log");
   using var store = new FasterKV<long, long>(1L << 20, new LogSettings { LogDevice = log });
-  using var s = store.NewSession(new SimpleFunctions<long, long>());
+  using var s = store.NewSession(new SimpleFunctions<long, long>((a, b) => a + b));
   long key = 1, value = 1, input = 10, output = 0;
   s.Upsert(ref key, ref value);
   s.Read(ref key, ref output);
   Debug.Assert(output == value);
   s.RMW(ref key, ref input);
-  s.RMW(ref key, ref input, ref output);
```

Comment:
There is no RMW method with such signature.

Reply:
This API was recently added, not yet in NuGet.

Reply:
Makes sense. Do you want me to revert this change to use the new API?

Reply:
Yes, you can revert it. We wanted this sample to show how RMW can now read the result of the operation with the new API, without requiring another Read.

Reply:
Makes sense 👍